src/app/(fullscreen)/(my-page)/withdraw/page.tsx

```
@@ -111,7 +112,7 @@ export default function WithDraw() {
             router.push(PATHS.EVENT);
           }}
           onCancel={undefined}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)